src/shared/util.js

```diff
@@ -86,6 +86,29 @@ const AnnotationType = {
   REDACT: 26,
 };
 
+const AnnotationStateModelType = {
```

Comment:
Any particular reason for not making all of these added structures enums?

Reply:
They are stored as strings in annotations, which is what I need to compare it against. Javascript can do string enums?

Reply:
Should I also order it alphabetically, or keep it together in 'logic' order like this? Move it somewhere ?

Reply:
I don't think the order matters much here, but it's usally nice to keep the `export` list at the bottom ordered so we can quickly spot if something is exported or not.